content/index.ja.md

```diff
@@ -0,0 +1,162 @@
+---
+# Translation instructions are after the "#" character in this first section. They are comments that do not show up in the web page. You do not need to translate the instructions after "#".
+# In this first section, do not translate the words before a colon. For example, do not translate "title:". Do translate the text after "title:".
+
+title: "WCAG 2 概要"
+nav_title: "Web Content – WCAG 2"
+lang: ja  # Change "en" to the translated-language shortcode
+last_updated: 2024-04-04  # Put the date of this translation YYYY-MM-DD (with month in the middle)
+
+# translators: # remove from the beginning of this line and the lines below: "# " (the hash sign and the space)
+# - name: "Makoto Ueki" # Add one -name: line for every translator
```

Comment:
```suggestion
# - name: "Makoto Ueki" # Add one -name: line for every translator
```

Please remove the hash sign and the space at the beginning of this line to make your name appear as translator.